### DISCUSSION TITLE
Error while I try to run an experiment from a csv file current (A)

### DISCUSSION THREAD

**Discussion by alihassan90** [pybamm-team/PyBaMM#3109] [General]
I get this error when i try to run a simple experiemnts from a csv file.

ValueError: drive_cycles should now be passed as an experiment step object, e.g. `pybamm.step.current(drive_cycle)`

My code looks like

drive_cycle=pd.read_csv("US06.csv",comment='#',header=None).to_numpy()
current_interpolant = pybamm.Interpolant(drive_cycle[:, 0], drive_cycle[:, 1], pybamm.t)
parameter_values["Current function [A]"] = current_interpolant

# =============================================================================
experiment=pybamm.Experiment(
     [
      ("Charge at 1A until 4.0 V",
       "Rest for 60 minutes",
       "Run US06_A (A)",
       )
      ],
     drive_cycles={
         "US06_A":drive_cycle,
         },  
     )
#

**Comment by valentinsulzer** [comment]
See https://github.com/pybamm-team/PyBaMM/blob/develop/examples/scripts/experiment_drive_cycle.py for the new syntax
